### ISSUE TITLE
Create the ability to Message

### ISSUE THREAD

**Issue by bdougie** [Bloc/Messenger#3]
The app currently does not give you the ability to create a new message thread and needs that to make this app a little more useful

**Comment by bdougie** [comment]
The problem lies in [the message post endpoint](https://www.bloc.io/apidoc/1.0/messages/create.html). It expects a symbol and JSON is sending a string.

```ruby
{
  user_id: 1,
  recipient_id: 2,
  subject: 'Royale with cheese'
  'stripped-text' => 'That's what they call a quarter pounder in France.'
}
```

**Comment by bdougie** [comment]
This was fixed in #12
